### PR TITLE
chore(medium-pass-2): Update all instances of "Medium Pass" name to be "Quick Assess"

### PR DESCRIPTION
**Pull request by madalynrose**
#### Details

This PR updates text, variable names, and file names to reference Quick Assess instead of Medium Pass.

##### Motivation

feature work 🚀 

#### Pull request checklist

- [n/a] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/tests/unit/tests/DetailsView/components/requirement-view-next-requirement-configuration.test.tsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ describe('RequirementViewNextRequirementConfiguration', () => {
         });
 
         it('returns Complete button when it is last requirement in mediumPassRequirementKeys', () => {
```
Comment (waabid) [inline src/tests/unit/tests/DetailsView/components/requirement-view-next-requirement-configuration.test.tsx:156]: nit: **mediumPass**RequirementKeys here (and above on line 144).

File: src/common/extension-telemetry-events.ts [hunk 2]
```diff
@@ -35,28 +35,29 @@ export const START_OVER_TEST: string = 'startOverTest';
 export const CANCEL_START_OVER_TEST: string = 'cancelStartOverTest';
 export const START_OVER_ASSESSMENT: string = 'startOverAssessment';
 export const CANCEL_START_OVER_ASSESSMENT: string = 'cancelStartOverAssessment';
-export const START_OVER_MEDIUM_PASS: string = 'startOverMediumPass';
-export const CANCEL_START_OVER_MEDIUM_PASS: string = 'cancelStartOverMediumPass';
+export const START_OVER_MEDIUM_PASS: string = 'startOverQuickAssess';
```
Comment (waabid) [inline src/common/extension-telemetry-events.ts:38]: The variables still refer to medium pass here.

**Review by waabid — COMMENTED** [review]
Changes look good! Just missing a couple of cases it seems!

File: src/tests/unit/tests/DetailsView/components/requirement-view-next-requirement-configuration.test.tsx (hunk 1 above)
Comment (madalynrose) [reply src/tests/unit/tests/DetailsView/components/requirement-view-next-requirement-configuration.test.tsx:156]: oh I thought I fixed this in the merge! good catch! fixing now!

File: src/common/extension-telemetry-events.ts (hunk 2 above)
Comment (madalynrose) [reply src/common/extension-telemetry-events.ts:38]: another good catch! thank you!

**Review by waabid — APPROVED** [review]

**Pull request merged**